include/mbedtls/asn1write.h

```diff
@@ -197,6 +197,28 @@ int mbedtls_asn1_write_ia5_string( unsigned char **p, unsigned char *start,
 int mbedtls_asn1_write_bitstring( unsigned char **p, unsigned char *start,
                           const unsigned char *buf, size_t bits );
 
+/**
+ * \brief           This function writes a named bitstring tag
+ *                  (#MBEDTLS_ASN1_BIT_STRING) and value in ASN.1 format.
+ *
+ *                  As stated in RFC 5280 Appendix B, trailing zeroes are
+ *                  omitted when encoding named bitstrings in DER.
+ *
+ * \note            This function works backwards within the data buffer.
+ *
+ * \param p         The reference to the current position pointer.
+ * \param start     The start of the buffer which is used for bounds-checking.
```

Comment:
I'm not a native, but I find this wording misleading because 'which' seems to apply to `buffer` here, and not `the start of`. I'd personally find `The start of the buffer, for bounds-checking.` clearer.

Reply:
I am not a native either, but this seems fine to me. Note that this was suggested by @sbutcher-arm... Also, I would rather not change this because then we have to go through all the review process again. In any case, its a really minor point...

Reply:
Ok.